shell/packages/sandstorm-db/db.js

```diff
@@ -391,6 +391,8 @@ ApiTokens = new Mongo.Collection("apiTokens");
 //              be considered revoked, and all live refs and sturdy refs obtained transitively
 //              through it must also become revoked. Each item is the JSON serialization of the
 //              `MembraneRequirement` structure defined in `supervisor.capnp`.
+//   hasApiHost: If true, there is an entry in ApiHosts for this token, which will need to be
```

Comment:
It seemed a little odd at first glance to have this information duplicated between the two tables, but I guess it saves a roundtrip in the token cleanup codepath, so that seems probably worthwhile.